docs/framework/wpf/app-development/wpf-windows-overview.md

```diff
@@ -140,7 +140,7 @@ Users interact with Windows Presentation Foundation (WPF) standalone application
   
  The window is finally opened by calling the <xref:System.Windows.Window.Show%2A> method; the result is shown in the following figure.  
   
- ![A Window Opened by Calling Window.Show](./media/windowoverviewfigure8.png "WindowOverviewFigure8")  
+ ![A Window Opened by Calling Window.Show](./media/wpf-windows-overview//window-opened-window-show.png)  
```

Comment:
A file name suggestion to avoid duplicate words

```suggestion
 ![A window opened by calling Window.Show](./media/wpf-windows-overview//window-opened-show-method.png)  
```